.github/workflows/build-wasm-examples.yml

```diff
@@ -7,13 +7,31 @@ env:
   PER_PAGE: 50
 
 jobs:
+  prepare-pages:
+    name: Get Pages
+    runs-on: ubuntu-latest
+    outputs:
+      pages: ${{ steps.pages.outputs.pages }}
+    steps:
+      - name: Checkout Bevy main branch
+        uses: actions/checkout@v4
+        with:
+          repository: 'bevyengine/bevy'
+          ref: 'latest'
+      - name: Get Pages
+        id: pages
+        run: |
+          example_count=`cat Cargo.toml | grep '\[\[example\]\]' | wc -l`
+          page_count=$((example_count / ${{ env.PER_PAGE }} + 1))
+          echo "pages=`python -c \"import json; print(json.dumps([i for i in range($page_count)]))\"`" >> $GITHUB_OUTPUT
```

Comment:
I'm not sure I like using `python` here. Could it perhaps use `jq`'s [`range`](https://jqlang.github.io/jq/manual/#range) function instead?